### DISCUSSION TITLE
Query What's the use of runner volume in Operator Deployment

### DISCUSSION THREAD

**Discussion by prary** [kiali/kiali#4374] [Q&A] [answered]
Hi,

I have a query for https://github.com/kiali/helm-charts/blob/abb34a83629945def72cf1db68ee6d9a586aa74b/kiali-operator/templates/deployment.yaml#L92 . Whats the purpose of runner volume in operator deployment?

I can see empty folder in operator pod.

**Comment by jmazzitelli** [comment, marked as answer]
I believe it is a place for the ansible runner to dump its logs:

https://github.com/kiali/helm-charts/blob/abb34a83629945def72cf1db68ee6d9a586aa74b/kiali-operator/templates/deployment.yaml#L51-L53

**Reply by prary** [reply]
Ohkay got it, after creating kiali resouce this folder got populated. 
Any idea how much size it can consume on node since it is mounted as emptyDir?
I understand it totally depends on the the number of kiali object created but any thoughts on memory usage of this folder would be extremely helpful? Is it possible to have timely clean-up if it reaches beyond certain size?

**Reply by jmazzitelli** [reply]
This is actually something that is used by the ansible operator SDK framework and not specifically by kiali itself  (the Kiali team did not add this nor use it explicitly -- this is only needed by the SDK framework itself).

I couldn't tell you an accurate answer to your questions. To get a good answer, you should ask the ansible operator SDK folks about this runner volume and how the operator framework uses it. For communicating with the operator SDK folks, see: https://github.com/operator-framework/community#get-involved
